model/host/host.go

```diff
@@ -92,6 +92,11 @@ type Host struct {
 
 	// accrues the value of idle time.
 	TotalIdleTime time.Duration `bson:"total_idle_time,omitempty" json:"total_idle_time,omitempty" yaml:"total_idle_time,omitempty"`
+
+	// managed containers require different information based on host type
+	Container string `bson:"container,omitempty" json:"container,omitempty"`
+	IsParent  bool   `bson:"is_parent,omitempty" json:"is_parent,omitempty"`
+	ParentId  string `bson:"parent_id,omitempty" json:"parent_id,omitempty"`
```

Comment:
We generally name things ID, as opposed to Id in newer code. The tags are fine as is.

Reply:
done